Homework2/palindrom.py

```diff
@@ -0,0 +1,22 @@
+# defines entered number - palindrom ot not.
+number = int(input("Введите число:"))
+kol_number = 0
+n = number
+while n > 0:
+    n = n // 10
+    kol_number += 1
+# divide our number into two parts
+# n = kol_number // 2
+# kol_number -= 1
+# comparing first part of number with second part, first digit with last digit
+for i in range(1, kol_number + 1, 2):
```

Comment:
there is more simple solution (you can use knowledge after Numeral system lection)